app/web/features/auth/login/Login.tsx

```diff
@@ -9,15 +9,12 @@ import { useRouter } from "next/router";
 import { useEffect } from "react";
 import vercelLogo from "resources/vercel.svg";
 import { dashboardRoute, signupRoute } from "routes";
-import makeStyles from "utils/makeStyles";
```

Comment:
Cleanup in a separate PR maybe. I know this is kinda harmless but again it's better to prevent bugs in case something needs to be reverted. Or to track changes that might have caused a bug.

Reply:
Moved to #4690